getting_started.md

```diff
@@ -373,32 +393,132 @@ There's a clear drop in diversity in the region of the selective sweep. And as e
+for such files:
+
+```{code-cell} ipython3
+import tskit
+
+ts.dump("data/my_tree_sequence.trees")
+new_ts = tskit.load("data/my_tree_sequence.trees")
+```
+
+It's also possible to export tree sequences to different formats. Note, however, that
+not only are these usually much larger files, but that analysis is usually much faster
+when performed by built-in tskit functions than by exporting and using alternative
+software. If you have a large tree sequence, you should *try to avoid exporting
+to other formats*.
+
+### Newick and Nexus format
+
+The most common format for interchanging tree data is Newick. 
+We can export to a newick format string quite easily. This can be useful for
+interoperating with existing tree processing libraries but is very inefficient for
+large trees. There is also no support for including sites and mutations in the trees.
+
+```{code-cell} ipython3
+small_ts = reduced_ts.keep_intervals([[0, 10000]])
+tree = small_ts.first()
+print(tree.newick(precision=3))
+```
+
+For an entire set of trees, you can use the Nexus file format, which acts as a container
+for a list of Newick format trees, one per line:
+
+```{code-cell} ipython3
+small_ts = small_ts.trim()  # Must trim off the blank region at the end of cut-down ts
+print(small_ts.to_nexus(precision=3))
+```
+
+### VCF
+
+The standard way of interchanging genetic variation data is the Variant Call Format, 
+for which tskit has basic support:
 
-:::{todo}
-Saving in VCF or ms format.
+```{code-cell} ipython3
+import sys
+small_ts.write_vcf(sys.stdout)
+```
+
+The write_vcf method takes a file object as a parameter; to get it to write out to the
+notebook here we ask it to write to stdout.
+
+:::{warning}
+The VCF output in tskit is somewhat primitive and has some gotchas. In particular,
+the mapping from floating point positions used in infinite sites mutations in msprime
+to integer POS values in VCF is done in a somewhat peculiar way.
 :::
 
+### Scikit-allel
+
+Because tskit integrates very closely with numpy, we can interoperate very efficiently
+with downstream Python libraries for working with genetic
+sequence data, such as [scikit-allel](https://scikit-allel.readthedocs.io/en/stable/).
+We can interoperate with {program}`scikit-allel` by exporting the genotype matrix as a
+numpy array, which {program}`scikit-allel` can then process in various ways.
+
+```{code-cell} ipython3
+import allel
+# Export the genotype data to allel. Unfortunately there's a slight mismatch in the 
+# terminology here where genotypes and haplotypes mean different things in the two
+# libraries.
+h = allel.HaplotypeArray(small_ts.genotype_matrix())
+print(h.n_variants, h.n_haplotypes)
+h
+```
+
+Sckit.allel has a wide-ranging and efficient suite of tools for working with genotype
+data, so should provide anything that's needed. For example, it gives us an
+alternative way to compute pairwise diversity:
```

Comment:
I don't think we mention diversity before this? Probably not worth putting this paragraph in at all, just showing how to get the data out is sufficient

Reply:
It's literally just in the section above, where we plot it out in a graph using the tskit version. So I think it's probably a good contract to leave it in.

Reply:
I stuck a link in to the place above where we used the native tskit version.